pkg/storegateway/gateway_blocks_http.go

```diff
@@ -74,6 +98,33 @@ func (s *StoreGateway) BlocksHandler(w http.ResponseWriter, req *http.Request) {
 	showDeleted := req.Form.Get("show_deleted") == "on"
 	showSources := req.Form.Get("show_sources") == "on"
 	showParents := req.Form.Get("show_parents") == "on"
+	actionType := req.Form.Get("action_type")
```

Comment:
because when it is empty, it would be default to none. just wanted to keep it as enum, do you mean we keep action as string and just hard coded == instead?